runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -3887,15 +3887,15 @@ void jitHookClassPreinitializeHelper(J9VMThread *vmThread,
    }
 
 
-/// This routine is used to indicate successful initialization of the J9Class
-/// before any Java code (<clinit>) is run. When analyzing code in the <clinit>
-/// with CHTable assumptions, this ensures that the CHTable is updated correctly.
-/// Otherwise a class will not be seen as having been initialized in the Java code
-/// reachable from the <clinit>; causing possibly incorrect devirtualization or other
-/// CHTable opts to be applied in a method called by <clinit> (if the <clinit> for class C calls a
-/// virtual method on an object of class C which is instantiated in the code reachable
-/// from <clinit> (this was an actual WSAD scenario)).
-///
+/** This routine is used to indicate successful initialization of the J9Class
+ before any Java code (<clinit>) is run. When analyzing code in the <clinit>
+ with CHTable assumptions, this ensures that the CHTable is updated correctly.
+ Otherwise a class will not be seen as having been initialized in the Java code
+ reachable from the <clinit>; causing possibly incorrect devirtualization or other
+ CHTable opts to be applied in a method called by <clinit> (if the <clinit> for class C calls a
+ virtual method on an object of class C which is instantiated in the code reachable
+ from <clinit> (this was an actual WSAD scenario)).
```

Comment:
May I ask you to add an asterisk at the start of each comment line in the block?